src/Planet.jl

```diff
@@ -42,7 +42,10 @@ export molmass_dryair,
     year_anom,
     orbit_semimaj,
     TSI,
-    MSLP
+    MSLP,
+    T_surf_ref,
+    ΔTv_ref,
```

Comment:
```suggestion
```
(As I suggested in the other PR, I'd remove `ΔTv_ref` here and instead replace this by `ΔTv_ref = T_surf_ref - T_min_ref` in the relevant code.)

Reply:
> in the relevant code.

In CLIMAParameters.jl? Or in ClimateMachine.jl